web/styles/subscriptions.css

```diff
@@ -842,7 +842,7 @@ h4.user_group_setting_subsection_title {
 
             .group-name,
             .sub-stream-name {
-                display: block;
+                display: inline;
                 width: 100%;
```

Comment:
`.sub-stream-name` is already a span element and hence `display: inline` applies to it by default. So let's just remove this line, along with `width: 100%` since that doesn't apply to `inline` elements.